packages/rum-recorder/src/domain/rrweb/record.ts

```diff
@@ -33,7 +33,7 @@ export function record(options: RecordOptions = {}): RecordAPI {
 
   const takeFullSnapshot = (isCheckout = false) => {
     const wasFrozen = mutationController.isFrozen()
-    mutationController.freeze() // don't allow any mirror modifications during snapshotting
+    mutationController.freeze() // don't allow any node to be serialized or mutation to be emitted during sharpshooting
```

Comment:
Could it be interesting to add some details about why freeze is needed?

Reply:
As I don't think 'freeze' is needed in the first place, I can't add details justifying its usage. Let's remove it in a future PR.